src/css/table.css

```diff
@@ -1,23 +1,59 @@
 table {
   display: grid;
```

Comment:
The columns line up a bit better when the display is left as the default. If you look at the stocktable css i set it to default and then set the width of the first column and it lined up alot better. Due to time constraints you could just try to set the first column width (i think i used 35%, probably different for each component) and that might help.